service/src/test/java/tds/config/services/impl/ConfigServiceImplIntegrationTests.java

```diff
@@ -0,0 +1,192 @@
+package tds.config.services.impl;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.ConfigServiceApplication;
+import tds.config.TimeLimits;
+import tds.config.services.ConfigService;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = ConfigServiceApplication.class)
+public class ConfigServiceImplIntegrationTests {
+    @Autowired
+    private ConfigService configService;
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // ClientTestProperty Tests
+    // -----------------------------------------------------------------------------------------------------------------
+    @Test
+    public void shouldGetAClientTestProperty() {
+        final String clientName = "SBAC_PT";
+        final String assessmentId = "SBAC Math 3-MATH-3";
+
+        Optional<ClientTestProperty> result = configService.getClientTestProperty(clientName, assessmentId);
+
+        assertThat(result.isPresent()).isTrue();
```

Comment:
Agreed - went through tests and updated to consistently use `isPresent()` or `isNotPresent()` where appropriate.